app/views/layouts/_google_analytics_script.haml

```diff
@@ -1,17 +1,17 @@
 - if (APP_CONFIG.use_google_analytics)
   <script type="text/javascript">
+  - if APP_CONFIG.use_google_tag_manager
```

Comment:
Environment variables are always strings, so string "false" will be a truthy value. So one thing you can do is to always convert to string and compare to "true"

For example: APP_CONFIG.use_google_tag_manager.to_s == "true"